### PR TITLE
Add http host tests v4

### PR DESCRIPTION
**Pull request by jmtaylor90**
Removed custom command as request in previous PR

### CODE REVIEW

File: tests/test-bad-http-host-rule-1/test.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 7.0.0
+
+  features:
+    - HAVE_LIBJANSSON
```
Comment (catenacyber) [inline tests/test-bad-http-host-rule-1/test.yaml:5]: By the way, `HAVE_LIBJANSSON` is implied by min version 7 ;-)

**Comment by catenacyber** [comment]
Looks good to me

File: tests/test-bad-http-host-rule-1/test.yaml (hunk 1 above)
Comment (jmtaylor90) [reply tests/test-bad-http-host-rule-1/test.yaml:5]: Okay, will update PR

**Comment by jmtaylor90** [comment]
Continued in https://github.com/OISF/suricata-verify/pull/887

**Pull request closed without merging**